package.json

```diff
@@ -13,11 +13,14 @@
     "url": "http://github.com/graphql/graphql-spec.git"
   },
   "scripts": {
-    "test": "spec-md spec/GraphQL.md > /dev/null",
+    "test": "npm run test:build && npm run test:spellcheck",
+    "test:build": "spec-md spec/GraphQL.md > /dev/null",
+    "test:spellcheck": "cspell 'spec/**/*.md'",
```

Comment:
Might as well spellcheck the README and the RFCs too?

Reply:
README yes. RFCs have a zillion spurious errors (like flagging github usernames) so I'll exclude those